vercel.json

```diff
@@ -364,14 +364,6 @@
       "source": "/reference/tools-and-interfaces/prisma-schema/models",
       "destination": "/docs/reference/tools-and-interfaces/prisma-schema/data-model"
     },
-    {
-      "source": "/getting-started/setup-prisma/add-to-existing-project",
-      "destination": "/docs/getting-started/setup-prisma/add-to-existing-project/relational-databases-typescript-postgres"
-    },
-    {
-      "source": "/getting-started/setup-prisma/start-from-scratch",
-      "destination": "/docs/getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-postgres"
-    },
```

Comment:
Removed intentionally?

Reply:
Yes, this is causing a lot of 308 redirect issues in the case of sidebar items.